server/main_test.go

```diff
@@ -34,6 +35,24 @@ func TestMain(m *testing.M) {
 	os.Exit(status)
 }
 
+type AssignedPrompt struct {
+	Adjectives []string `json:"adjectives"`
+	Noun       string   `json:"noun"`
+}
+
+type CurrentPlayer struct {
+	AssignedPrompt *AssignedPrompt `json:"assignedPrompt"`
+	IsHost         bool            `json:"isHost"`
+	Name           string          `json:"name"`
+}
+
+type GameStateResponse struct {
```

Comment:
can we import this directly from the state_manager instead of duplicating this struct?

Reply:
How about we move this to state manager and stop returning raw models and return these instead?

Reply:
e.g. one can be `InternalGameState` and the one we send back down to clients just `GameState`. `InternalGameState` can become an ORM-ey model in the future and `GameState` could become a proto model. What do you think?

Reply:
++ we should definitely separate the storage format from the wire format. i dont think its critical while we prototype but maybe we should start keeping a list of v1 features. ill add a meta issue to keep track of that stuffs.

Reply:
#37